jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/util/Util.java

```diff
@@ -202,4 +209,74 @@ public static List<String> retrieveErrorRecords(String responseStr){
 		
 		return errorList;
 	}
+	
+	  /**
+     * This is inspired by java hash function.
+     *
+     * @param inStr the in str
+     * @return the unique id for string
+     */
+    public static String getUniqueIdForString(String inStr) {
+        MessageDigest md;
+        try {
+            md = MessageDigest.getInstance("MD5");
+        } catch (NoSuchAlgorithmException e) {
+            // if algorithm does not exist, fall back and try to generate unique
+            // hash
+            LOGGER.error("unable to generate has usnig Md5", e);
+            LOGGER.error("falling back to hash generation");
+            return hash(inStr);
+        }
+        md.update(inStr.getBytes());
+        byte byteData[] = md.digest();
+        // convert the byte to hex format method 2
+        StringBuffer hexString = new StringBuffer();
+        for (int i = 0; i < byteData.length; i++) {
+            String hex = Integer.toHexString(0xff & byteData[i]);
+            if (hex.length() == 1)
+                hexString.append('0');
+            hexString.append(hex);
+        }
+        return hexString.toString();
+    }
+    
+    /**
+     * Hash.
+     *
+     * @param s the s
+     * @return the string
+     */
+    public static String hash(String s) {

```

Comment:
Copy-paste again

Reply:
removed above function. it is not used any more